fastplotlib/layouts/_gridplot.py

```diff
@@ -360,8 +365,18 @@ def show(
                 0, 0
             ].camera.maintain_aspect
 
+        # validate vbox if not None
+        if vbox is not None:
+            for widget in vbox:
+                if not isinstance(widget, Widget):
+                    raise ValueError(f"Items in vbox must be ipywidgets. Item: {widget} is of type: {type(widget)}")
+            self.vbox = VBox(vbox)
+
         if not sidecar:
-            return VBox([self.canvas, self.toolbar.widget])
+            if self.vbox is not None:
+                return VBox([self.canvas, self.toolbar.widget, self.vbox])
```

Comment:
from the previous comment, I just realized that we can't use `VBox` in `show()` because it'll raise an error if using in an environment without `ipywidgets`. Perhaps we need to split `show()` into `show_desktop()` and `show_notebook()`?

Or have two subclasses of `PlotFrame`, one for notebook and one for desktop, I think this will be the cleaner solution.